### DISCUSSION TITLE
Modification of contract notice in the Dynamic Purchasing System

### DISCUSSION THREAD

**Discussion by EvijaMozga** [OP-TED/eForms-SDK#1056] [Q&A] [answered]
Dear TED Team,

If the rules of the DPS are amended, for example, by changing the DPS deadline after the deadline for submitting requests to participate has passed, and the contract notice is modified rather than a new procurement procedure is announced (by publishing a new notice), is it permissible (technically feasible) to set a new deadline for submitting requests to participate if the previous deadline has already expired?

Thank you for your support in advance!

Kind regards,

Evija Mozga
Procurement Monitoring Bureau

**Comment by YvesJo** [comment]
Hi,
I guess you are referring to a Change to an existing Competition Notice. To be able to apply a change, it will be technically necessary to have the request to participate submission deadline later the dispatch date.
I have strong doubts that modifying an existing competition notice while deadlines have passed would be a good way to proceed just to not start a new procedure and invite you to get in touch with the helpdesk regarding that question.

**Reply by EvijaMozga** [reply]
Dear TED Team,

Thank you for your answer.
We have consulted the EC on this issue and have received the following answer: _There is no deadline for submission of request participate since they can be submitted throughout the whole duration of DPS (art. 34(1)). The time limit indicated in art. 34 second sub-paragraph (a) is the time limit for submitting request to participate before the first invitation tender is sent. But request to participate can be submitted also after (just with a risk of missing some earlier sent invitations, if any). Thus, validity of DPS can be extended any time before its expire and request to participate may also be submitted at any time before its expiry._
Taking into account both answers, in a case of extending the validity of DPS, what is the correct way to announce it: by publishing **change to the existing Competition notice** or by publishing **new Competition notice** without indicating deadline for submission of requests to participate? 
And will it be possible to publish any notice from the technical point of view if for extending the validity of DPS there is no deadline, however, the request to participate submission deadline must be indicated later than the dispatch date? Is this validation rule in accordance with the answer from the EC?
Thank you in advance for your time!

Kind regards,

Evija Mozga
Procurement Monitoring Bureau

**Reply by YvesJo** [reply]
Hi,
After a closer look at the specific case, the rule BR-BT-00630-0150 that required the Deadline for Request to participate (BT-630) to be later than the Notice Dispatch Date (BT-05) has been modified with SDK 1.11 and does not apply anymore to a Change notice; as a consequence for the case in question it is recommended to use a Change notice.
Sorry for the confusion.
KR.

**Reply by EvijaMozga** [reply]
Thank you for your reply!
Does it mean that the deadline for request to participate can be left out or the previous (historical) deadline must be provided in the case of extending the validity of DPS and publishing a Change notice for the contract notice that established the DPS?
Thank you in advance!

Kind regards,

Evija Mozga
Procurement Monitoring Bureau

**Reply by YvesJo** [reply]
Hi,
Rules on the existence will still need to be verified, so as long as the field is required, information will need to be present.
I saw that I also confused two Business Terms BT-630 and BT-1311. I couldn't find though any applied rule on BT-1311 that would require it to be later than the submission deadline. I shall go for clarification about BT-1311 to see whether this should remain with the initial value, which could be confusing in the change notice.
I will rais ethis concern and will get back to you.

**Reply by EvijaMozga** [reply]
Thank you. Regarding this - I couldn't find though any applied rule on BT-1311 that would require it to be later than the submission deadline - these fields in the notice are never indicated simultaneously. The request receipt deadline is specified in two-stage procedures, while the bid submission deadline is indicated in the invitation sent to the selected candidates, but is not mentioned in the notice; in one-stage procedures, the tender submission deadline is specified, while the request receipt deadline is not indicated.

**Reply by YvesJo** [reply]
sorry, I meant notice submission date (i.e. notice dispatch date). However some questions still subsist and I wish to have these clarified first to detect if any rule update would be needed and what actual data is expect in a change.

**Reply by YvesJo** [reply]
Hi,
following the first collected feedback, a change has to be used and the request to participate deadline may have to be updated to match the end of the DPS so that this will continue to appear as part of the open opportunities. Technically for the change there is no restriction on the deadline, even it is passed. Deadline for request for information could also need some update.
I'm however waiting for some confirmation and this could occur as part of a broader activity on DG GROW side about DPS.
I'll put some additional information here as soon as I have further information.
KR

**Reply by EvijaMozga** [reply]
Dear TED Team,

We wanted to kindly ask if there is any additional information on this issue?

Kind regards,

Evija Mozga
Procurement Monitoring Bureau

**Reply by YvesJo** [reply, marked as answer]
Hi,
Nothing new so far. In case of business questions, you could may be ask GROW directly. 
If technical restrictions would prevent from doing what is needed, then report your concerns here.
KR
